Spec/RealtimeClientConnection.swift

```diff
@@ -245,7 +247,7 @@ class RealtimeClientConnection: QuickSpec {
                             done()
                         case .connected:
                             if let transport = client.internal.transport as? TestProxyTransport, let query = transport.lastUrl?.query {
-                                expect(query).to(haveParam("lib", withValue: "cocoa\(ARTDefault_variant)-1.2.4"))
+                                expect(query).to(haveParam("lib", hasPrefix: "\(ARTDefault_libName)/1.2."))
```

Comment:
I know you based this on the previous test boilerplate, but I would prefer if you test against a static string (i.e. don't make the test rely on `ARTDefault_libraryName` being correctly assigned).

There is another instance of this that also needs replacing, please.

Reply:
https://github.com/ably/ably-cocoa/pull/1119/commits/abbad0642963b0c8b3f8d11fe78be014fe4a478e